generatedocs/data/params.json

```diff
@@ -456,6 +459,12 @@
     "randomaction.actions.overrides": [
         "(optional) Static overrides to the action. The overrides can include any or all of the settings from the original action, as determined by the `type` field. If nothing is specified, the default values are used."
     ],
+    "reconnectsettings.reconnect" : [
+        "Enable reconnect attempt when websocket gets disconnected, defaults to false."
+    ],
+    "reconnectsettings.backoff" : [
+        "Reconnect backoff scheme, defaults to [0.0, 1.0, 10.0, 20.0] if left empty"
```

Comment:
Suggestion (add explanation of numbers as bullets):
    "reconnectsettings.backoff" : [
        "Re-connection backoff scheme. Defaults to `[0.0, 1.0, 10.0, 20.0]`, if left empty",
			"`0.0`: If the WebSocket is disconnected, wait 0.0s before attempting to re-connect",
			"`1.0`: If the previous attempt to re-connect failed, wait 1.0s before attempting again",
			"`10.0`: If the previous attempt to re-connect failed, wait 10.0s before attempting again",
			"`20.0`: If the previous attempt to re-connect failed, wait 20.0s before attempting again"
    ],